### PR TITLE
[tidy] smallest model first

### PR DESCRIPTION
**Pull request by thenatlog**
**Make the default model an even smaller model #103**  

- removes select logic which incorrectly selected the third element in the model list as default

### CODE REVIEW

**Review by AlexCheema — APPROVED** [review]
Please fix indentation to 2 space indent then I will review

**Review by AlexCheema — CHANGES_REQUESTED** [review]
Please fix indentation to 2 space indent then I will review

File: exo/tinychat/index.html [hunk 1]
```diff
@@ -29,8 +29,8 @@
     <div x-show="errorMessage" x-transition.opacity x-text="errorMessage" class="toast">
     </div>
 <div class="model-selector">
-<select @change="if (cstate) cstate.selectedModel = $event.target.value" x-model="cstate.selectedModel">
```
Comment (AlexCheema) [inline exo/tinychat/index.html:32]: don't we still need this to set the selectedModel

File: exo/tinychat/index.html [hunk 2]
```diff
@@ -29,8 +29,8 @@
...
-<option selected="" value="llama-3.2-1b">Llama 3.2 1B</option>
+<select>
+<option value="llama-3.2-1b">Llama 3.2 1B</option>
```
Comment (AlexCheema) [inline exo/tinychat/index.html:33]: shouldn't this option have a `selected` property so that it's by default selected

**Review by AlexCheema — CHANGES_REQUESTED** [review]

File: exo/tinychat/index.html (hunk 2 above)
Comment (thenatlog) [reply exo/tinychat/index.html:33]: the first item in the list is selected by default in <select> tags

File: exo/tinychat/index.html (hunk 1 above)
Comment (AlexCheema) [reply exo/tinychat/index.html:32]: @thenatlog

**Pull request merged**